packages/hooks/use-fills.tsx

```diff
@@ -0,0 +1,39 @@
+import { useConnect } from "frontity";
+
+interface Fills {
+  state: {
+    fills: {
+      [key: string]: {
+        slot: string;
+        library: string;
+        priority: number;
+        props: {
+          [key: string]: any;
+        };
+      };
+    };
+  };
+  libraries: {
+    [key: string]: any;
+  };
+}
+
+const useFills = (name) => {
+  const { state, libraries } = useConnect<Fills>();
```

Comment:
@orballo Could you tell me if this is the right approach to typing the output of `useConnect` ?

Reply:
That's the right way